python/ml4ir/applications/ranking/tests/test_fixed_additive_positional_bias.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+import warnings
+import tensorflow as tf
+from ml4ir.base.model.architectures.fixed_additive_positional_bias import FixedAdditivePositionalBias
+
+warnings.filterwarnings("ignore")
+
+
+class TestFixedAdditivePositionalBias(unittest.TestCase):
+
+    def calling_additive_positional_bias(self, rank_index, max_ranks, training):
+        """Testing to invoke additive positional bias"""
```

Comment:
This is a minor comment, feel free to ignore if you think is of no added value. We should improve as a team on what we write and how we write docstrings. If we are to take it one step at a time, we should at least use action verbs..
For instance, the name of this function and the docstring. I think it would make more sense to name them `call_additive_positional_bias` or even better `apply_additive_positional_bias`. Accordingly the dosctring probably needs a verb: `Tests if the positional bias is applied as expected during training (`training=True`) or evaluation`.